components/onboarding-modal/OnboardingModal.js

```diff
@@ -258,7 +258,7 @@ class OnboardingModal extends React.Component {
     const { step, isSubmitting, error, noOverlay } = this.state;
 
     return (
-      <React.Fragment>
+      <React.Fragment data-cy="collective-modal">
```

Comment:
`Fragment`s are not included in the DOM, is this `data-cy` intended?

Reply:
You are right, That was not intended, Removed